openrpc.json

```diff
@@ -0,0 +1,4302 @@
+{
+	"openrpc": "1.2.4",
```

Comment:
Can we move this to live in the folder with the tests? And maybe run it through a minimizer?

Reply:
Yes, for sure, we can do that.
Also, we can try compiling this file from source just before the test: https://github.com/ethereum/execution-apis